dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -3257,7 +3267,7 @@ paths:
         schema:
           type: "array"
           items:
-            $ref: "#/definitions/SourceFile"
+            type: "object"
```

Comment:
Is this part of the PR?

Reply:
This happens when I mvn clean install locally.